notebooks/poisoning_attack_sleeper_agent_pytorch.ipynb

```diff
@@ -0,0 +1,2150 @@
+{
```

Comment:
Please update `notebooks/README.md` with this new notebook.

Reply:
Sure

Reply:
Done